packages/cashier/src/pages/on-ramp/__tests__/on-ramp.spec.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import { fireEvent, render, screen } from '@testing-library/react';
 import { isMobile, routes } from '@deriv/shared';
 import OnRamp from '../on-ramp';
+import { StoreProvider } from '../../../hooks';
+import { TRootStore, DeepPartial } from '../../../types';
```

Comment:
Making an alias for types directory in your webpack config is a good approach to avoid these long imports.

Reply:
The alias has been added in other PR and all the imports will be changed to alias when [this](https://github.com/binary-com/deriv-app/pull/6281) PR gets merged.